start/routes.ts

```diff
@@ -1,25 +1,11 @@
-/*
-|--------------------------------------------------------------------------
-| Routes
-|--------------------------------------------------------------------------
-|
-| This file is dedicated for defining HTTP routes. A single file is enough
-| for majority of projects, however you can define routes in different
-| files and just make sure to import them inside this file. For example
-|
-| Define routes in following two files
-| ├── start/routes/cart.ts
-| ├── start/routes/customer.ts
-|
-| and then import them inside `start/routes.ts` as follows
-|
-| import './routes/cart'
-| import './routes/customer'
-|
-*/
-
 import Route from '@ioc:Adonis/Core/Route'
 
-Route.get('/', async () => {
-  return { hello: 'world' }
-})
+Route.group(() => {
+  // Authentication routing
+  Route.post('/login', 'Api/V1/AuthController.create')
+  Route.delete('/logout', 'Api/V1/AuthController.destroy').middleware('auth')
+
+  // Registration routing
+  Route.post('/registrations', 'Api/V1/RegistrationsController.store')
+  Route.delete('/registrations', 'Api/V1/RegistrationsController.destroy').middleware('auth')
+}).prefix('/api/v1')
```

Comment:
Ça me semble plus scallable comme ça. Ici tu n'as que l'authentification, mais par la suite tu auras d'autres routes.

```suggestion
Route.group(() => {
  // public routes
  Route.group(() => {
    Route.post('/login', 'Api/V1/AuthController.create')
    Route.post('/registrations', 'Api/V1/RegistrationsController.store')
  })
  
  // authenticated routes
  Route.group(() => {
    Route.delete('/logout', 'Api/V1/AuthController.destroy')
    Route.delete('/registrations', 'Api/V1/RegistrationsController.destroy')
  }).middleware('auth')
}).prefix('/api/v1')
```